content/en/about/feature-stages/index.md

```diff
@@ -69,17 +69,19 @@ Below is our list of existing features and their current phases. This informatio
 
 | Feature           | Phase
 |-------------------|-------------------
-| [Deny Checker](/docs/reference/config/policy-and-telemetry/adapters/denier/)         | Stable
-| [List Checker](/docs/reference/config/policy-and-telemetry/adapters/list/)        | Stable
-| [Pluggable Key/Cert Support for Istio CA](/docs/tasks/security/plugin-ca-cert/)        | Stable
 | [Service-to-service mutual TLS](/docs/concepts/security/#mutual-tls-authentication)         | Stable
 | [Kubernetes: Service Credential Distribution](/docs/concepts/security/#pki)   | Stable
+| [SDS based key and certificate provisioning](/docs/concepts/security/#the-istio-agent-in-kubernetes)  | Stable
```

Comment:
fyi, this link is now broken after https://github.com/istio/istio.io/pull/6727 (The Istio Agent in Kubernetes is removed)
What do you suggest for replacement? @myidpt

Reply:
This line is going to be removed. I think it is already covered in [Kubernetes: Service Credential Distribution]. @ericvn will remove it.